install.sh

```diff
@@ -75,6 +75,8 @@ install_nvm_from_git() {
       exit 1
     }
   else
+    echo "=> Cleanup old nvm dir '$INSTALL_DIR'"
+    command rm -rf $INSTALL_DIR/* $INSTALL_DIR/.*
```

Comment:
What about backup the installed node versions and their modules first and restore after that?